tests/e2e/performance/test_io_performance.py

```diff
@@ -73,7 +69,8 @@ def test_run_io(
         logging.info("IOPs after FIO:")
         reads = fio_result.get('jobs')[0].get('read').get('iops')
         writes = fio_result.get('jobs')[0].get('write').get('iops')
+        w_bw = fio_result.get('jobs')[0].get('write').get('bw')
+        r_bw = fio_result.get('jobs')[0].get('read').get('bw')
         logging.info(f"Read: {reads}")
         logging.info(f"Write: {writes}")
-        g_sheet = GoogleSpreadSheetAPI("OCS FIO", sheet_index)
```

Comment:
why we removed saving it in google doc, it might still worth to keep it there?

Reply:
this test is an old one and I don't think anyone even looking at the google doc for this test.

Reply:
I think its still worth keeping this in gsheet, the graph plotting is easier here even if we use once in while.

Reply:
can be taken up later.